packages/pipeline-editor/src/PipelineEditor/index.tsx

```diff
@@ -312,14 +312,15 @@ const PipelineEditor = forwardRef(
                 "filehandler"
               );
               const parameters = e.targetObject.app_data?.component_parameters;
+
               return [
                 {
                   action: "openFile",
                   label: "Open File",
                   // NOTE: This only checks if the string is empty, but we
                   // should verify the file exists.
                   enable:
-                    filenameRef &&
+                    filenameRef !== undefined &&
                     parameters?.[filenameRef] !== undefined &&
                     parameters?.[filenameRef].trim() !== "",
```

Comment:
Can we wrap this all in a double bang so it will always resolve to a boolean? 
```ts
!!(
  filenameRef && 
  parameters?.[filenameRef] !== undefined &&
  parameters?.[filenameRef].trim() !== ""
)
```